### PR TITLE
Custom select

### PR DESCRIPTION
**Pull request by GneyHabub**
Custom selects. The API is similar to inputs: 
 1. Provide `name` to register it in the react-hook-form, although the validation must be done through an external resolver.
 2. There are two sizes - M and L
 3. Pass the `isLive` prop to add this red dot on the left to indicate that the change of the select will have an immediate effect.

### CODE REVIEW

File: kafka-ui-react-app/src/components/common/Select/LiveIcon.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      </svg>
+    </i>
+  );
+};
+
+export default styled(LiveIcon)`
+  position: absolute;
+  left: 12px;
+  top: 50%;
+  transform: translateY(-36%);
```
Comment (abelgibayev) [inline kafka-ui-react-app/src/components/common/Select/LiveIcon.styled.tsx:30]: Why `-36%`?
I think that it should be -50%. (https://www.figma.com/file/ZkvysdRUmt9p2PDNU3a9WU/Kafka-UI?node-id=64%3A3950)

File: kafka-ui-react-app/src/components/common/Select/LiveIcon.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import React from 'react';
+
+interface Props {
+  className?: string;
+  selectSize: 'M' | 'L';
```
Comment (abelgibayev) [inline kafka-ui-react-app/src/components/common/Select/LiveIcon.styled.tsx:6]: The size is unused

**Review by abelgibayev — APPROVED** [review]

File: kafka-ui-react-app/src/components/common/Select/LiveIcon.styled.tsx (hunk 1 above)
Comment (GneyHabub) [reply kafka-ui-react-app/src/components/common/Select/LiveIcon.styled.tsx:30]: I'm not sure how it works, but 50% doesn't make centered:
image

And 36 does:
image

Comment (abelgibayev) [reply kafka-ui-react-app/src/components/common/Select/LiveIcon.styled.tsx:30]: try to add `line-height: 1;`.

**Pull request merged**